### PR TITLE
changes .env to jungle tesnet

### PR DESCRIPTION
**Pull request by fagomezra**

### CODE REVIEW

**Review by AngeloCG97 — APPROVED** [review]
LGTM

File: .github/workflows/push-dev-environment.yaml [hunk 1]
```diff
@@ -488,14 +488,14 @@ jobs:
...
-                  NEXT_PUBLIC_EOS_READ_RPC_URLS: https://api.waxtest.alohaeos.com,https://wax-testnet.cryptolions.io,https://testnet.wax.pink.gg,https://testnet.waxsweden.org
+                  NEXT_PUBLIC_EOS_READ_RPC_URLS: https://jungle4.api.eosnation.io
                   NEXT_PUBLIC_TABLE_ROWS_MAX_FETCH_PER_SEC: 10
-                  NEXT_PUBLIC_EDEN_CONTRACT_ACCOUNT: test.edev
+                  NEXT_PUBLIC_EDEN_CONTRACT_ACCOUNT: genesisdeden
                   NEXT_PUBLIC_AA_FETCH_AFTER: 1633883520000
                   NEXT_PUBLIC_TOKEN_CONTRACT: eosio.token
-                  NEXT_PUBLIC_TOKEN_SYMBOL: WAX
+                  NEXT_PUBLIC_TOKEN_SYMBOL: EOS
                   NEXT_PUBLIC_TOKEN_PRECISION: 8
```
Comment (leisterfrancisco) [inline .github/workflows/push-dev-environment.yaml:499]: 4

File: .github/workflows/push-dev-environment.yaml [hunk 2]
```diff
@@ -506,7 +506,7 @@ jobs:
                   NEXT_PUBLIC_AA_SCHEMA_NAME: members
                   NEXT_PUBLIC_BLOCKEXPLORER_ACCOUNT_BASE_URL: https://wax-test.bloks.io/account
                   NEXT_PUBLIC_BLOCKEXPLORER_TRANSACTION_BASE_URL: https://wax-test.bloks.io/transaction
-                  NEXT_PUBLIC_APP_MINIMUM_DONATION_AMOUNT: 10.00000000 WAX
+                  NEXT_PUBLIC_APP_MINIMUM_DONATION_AMOUNT: 10.00000000 EOS
```
Comment (leisterfrancisco) [inline .github/workflows/push-dev-environment.yaml:509]: 10.0000

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by leisterfrancisco — APPROVED** [review]

**Pull request merged**